release-content/0.16/migration-guides/_guides.toml

```diff
@@ -82,12 +82,6 @@ prs = [17967]
 areas = ["Cross-Cutting"]
 file_name = "17967_Upgrade_to_Rust_Edition_2024.md"
 
-[[guides]]
-title = "Don't reëxport `bevy_image` from `bevy_render`"
-prs = [16163]
-areas = ["Cross-Cutting", "Rendering"]
-file_name = "16163_Dont_reëxport_bevy_image_from_bevy_render.md"
-
```

Comment:
This is a 0.15 change: https://bevyengine.org/learn/migration-guides/0-14-to-0-15/#don-t-re-export-bevy-image-from-bevy-render